src/plugins/dashboard/public/plugin.tsx

```diff
@@ -78,7 +78,7 @@ export class DashboardEmbeddableContainerPublicPlugin
   ): Setup {
     const expandPanelAction = new ExpandPanelAction();
     uiActions.registerAction(expandPanelAction);
-    uiActions.attachAction(CONTEXT_MENU_TRIGGER, expandPanelAction);
+    uiActions.attachAction(CONTEXT_MENU_TRIGGER, expandPanelAction.id);
```

Comment:
Because on line 80 `uiActions.registerAction(expandPanelAction)` the action is already registered. And line 81 just attaches action to a trigger.

`addTriggerAction` can also be used there, the only difference is that `addTriggerAction` will first try to register an action, which is already registered.